python/tk_vred/menu_generation.py

```diff
@@ -20,7 +20,7 @@ class VREDMenuGenerator(object):
     Menu generation functionality for VRED.
     """
 
-    ROOT_MENU_TEXT = u"S&hotGrid"
+    ROOT_MENU_TEXT = "S&hotGrid"
```

Comment:
Just checking that we meant to remove the `u` prefix (which makes the menu string unicode) and still works in VRED?

Reply:
@staceyoue The `black` pre-commit formatter took it out - good spotting!
I just launched a handful of versions back through 2020.3 of VRED and the menu appears so it seems to be okay.
Thanks!